src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/AbstractClassNameCheck.java

```diff
@@ -34,7 +34,7 @@
  * <p>
  * Rationale: Abstract classes are convenience base class implementations of
  * interfaces, not types as such. As such they should be named to indicate this.
- * Also, if names of classes starts with 'Abstract' it's very convenient that
+ * Also, if the names of classes start with 'Abstract' it's very convenient that
  * they will have abstract modifier.
```

Comment:
`As such they should be named to indicate this. Also, if the names of classes start with 'Abstract' it's very convenient that they will have abstract modifier.` -> `For this reason, it should be made obvious that a given class is abstract by prefacing the class name with 'Abstract'.`

Reply:
done